src/App.css

```diff
@@ -49,11 +50,16 @@ span {
   background-color: #88a65e;
   height: fit-content;
   text-align: center;
+  cursor: pointer;
   padding: 10px;
+  border-radius: 5px;
 }
 #categories a {
     color: white;
```

Comment:
add `cursor: pointer` to categories as well?

Reply:
although, I don't think `a` works when it's a list, it's `.item` now I believe

Reply:
seems to get the color, which is the end result